### PR TITLE
API responsiveness SLO: move filtering to Prometheus

### PR DESCRIPTION
**Pull request by oxddr**
API responsiveness SLO: move filtering to Prometheus. 

/assign @krzysied

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go [hunk 1]
```diff
@@ -40,10 +39,14 @@ import (
 const (
 	apiResponsivenessPrometheusMeasurementName = "APIResponsivenessPrometheus"
 
+	filters = `resource!="events", verb!~"WATCH|WATCHLIST|PROXY|proxy|CONNECT"`
+
 	// latencyQuery %v should be replaced with query window size.
```
Comment (krzysied) [inline clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go]: nit: Currently there are two %v in the query. Comment should be updated.

File: clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go [hunk 2]
```diff
@@ -40,10 +39,14 @@ import (
...
-	latencyQuery = "quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile[%v])"
+	// TODO(krzysied): figure out why we're getting non-capitalized proxy and fix this.
+	latencyQuery = "quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile{%v}[%v])"
+
 	// countQuery %v should be replaced with query window size.
```
Comment (krzysied) [inline clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go]: Same comment as above.

**Comment by krzysied** [comment]
/lgtm
/approve

Thanks!

**Comment by mm4tt** [comment]
Nice!

**Comment by wojtek-t** [comment]
nice!

**Pull request merged**